.github/workflows/alternative-ci-runners-4.yml

```diff
@@ -13,37 +13,41 @@ permissions:
   contents: read
 
 jobs:
-  docs-lint-on-northflank-kata-remote-engine:
+  docs-lint-on-nf-type-01-0-18-7-remote-engine:
     uses: ./.github/workflows/_dagger_on_northflank_remote_engine.yml
     with:
-      machine: nf-kata-az-amd-qemu
+      machine: nf-type-01-0.18.7
       function: check --targets=docs
 
-  sdk-go-on-northflank-kata-remote-engine:
-    needs: docs-lint-on-northflank-kata-remote-engine
+  sdk-go-on-nf-type-01-0-18-7-remote-engine:
+    needs: docs-lint-on-nf-type-01-0-18-7-remote-engine
     uses: ./.github/workflows/_dagger_on_northflank_remote_engine.yml
     with:
-      machine: nf-kata-az-amd-qemu
+      machine: nf-type-01-0.18.7
       function: check --targets=sdk/go
 
-  sdk-python-on-northflank-kata-remote-engine:
-    needs: docs-lint-on-northflank-kata-remote-engine
-    uses: ./.github/workflows/_dagger_on_northflank_remote_engine.yml
-    with:
-      machine: nf-kata-az-amd-qemu
-      function: check --targets=sdk/python
+#  docs-lint-on-nf-type-02-0-18-7-remote-engine:
+#    uses: ./.github/workflows/_dagger_on_northflank_remote_engine.yml
+#    with:
+#      machine: nf-type-02-0.18.7
+#      function: check --targets=docs
+#
+#  sdk-go-on-nf-type-02-0-18-7-remote-engine:
+#    needs: docs-lint-on-nf-type-02-0-18-7-remote-engine
+#    uses: ./.github/workflows/_dagger_on_northflank_remote_engine.yml
+#    with:
+#      machine: nf-type-02-0.18.7
+#      function: check --targets=sdk/go
```

Comment:
I'm a fan of keeping things clean & removing this. We can always add it back when we are ready to re-enable it.

Reply:
Sure, i will drop those as well.